.github/workflows/publish.template.yaml

```diff
@@ -29,6 +41,12 @@ jobs:
         run: npm ci && npm run build
         working-directory: plugins/${{ inputs.affected-plugin }}
 
+      - name: Publish version dry-run
+        run: npm publish --access public --tag ${{ inputs.version-tag && 'beta' || 'latest' }} --dry-run
```

Comment:
`${{ inputs.version-tag && 'beta' || 'latest' }}` this seems a bit strange as the input being a string a user might pass `version-tag=alpha`